startup/10-detectors.py

```diff
@@ -683,9 +743,13 @@ def collect(self):
 
             chunk_count = linecount // self.chunk_size + int(linecount % self.chunk_size != 0)
             for chunk_num in range(chunk_count):
-                datum_uid = self._reg.register_datum(self.resource_uid,
-                                                     {'chunk_num': chunk_num,
-                                                      'column' : self._column})
+        
+                datum_id = '{}/{}'.format(self._resource_uid,  next(self._datum_counter))
+                datum = {'resource': self._resource_uid,
+                         'datum_kwargs': {{'chunk_num': chunk_num,
+                                           'column' : self._column}},
```

Comment:
Why is this block in double curly braces? I don't think it will work as expected:
```py
In [1]: d = {{'foo': 'bar'}}
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-f6f92fa53f40> in <module>
----> 1 d = {{'foo': 'bar'}}

TypeError: unhashable type: 'dict'
```